### PR TITLE
Musark 682

### PR DESCRIPTION
**Pull request by rituvesh**

### CODE REVIEW

File: language.json [hunk 1]
```diff
@@ -7,6 +7,18 @@
         "loggedIn": "Logged in as %{name}",
         "logout": "Logout",
         "collection": "Collection",
+        "collections": {
+          "Alger": "Algae",
```
Comment (jarlah) [inline language.json]: This is the exact reason why i did not do it like this. This is an antipattern. You can remove the Norwegian translation if it doesn't do anything with the keystring. E.g. If it doesn't remove the spaces.

Comment (jarlah) [reply language.json]: But i really really dislike it.

Comment (jarlah) [reply language.json]: U should actually check if there is a translation for the name. If there is no translation use the name without translating it.

**Review by jarlah — CHANGES_REQUESTED** [review]
Need to fix up this a bit.

File: language.json (hunk 1 above)
Comment (jarlah) [reply language.json]: And stop using manes AS keys. Use UUID.

**Comment by rituvesh** [comment]
Added the UUID of collections in language keys. 

Note: In future if we have to add collections then we must use same UUID on all the instances of database.

**Review by jarlah — APPROVED** [review]

**Review by jarlah — APPROVED** [review]

**Pull request merged**